corehq/apps/app_manager/models.py

```diff
@@ -1710,7 +1715,7 @@ def custom_icon(self):
             return self.custom_icons[0]
 
 
-class Form(IndexedFormBase, NavMenuItemMediaMixin):
+class Form(IndexedFormBase, FormMediaMixin, NavMenuItemMediaMixin):
```

Comment:
Would these media mixins be better on the `IndexedFormBase` or `FormBase` classes? Actually I'm not even sure why we need both of those base classes in general

Reply:
Now that you mention it, I don't know why we have both `FormBase` and `IndexedFormBase` either, or why `NavMenuItemMediaMixin` isn't mixed in to one of those instead of where it is. I'll take a stab at simplifying the class hierarchy, but will do it in a followup PR.